src/server/svr_chk_owner.c

```diff
@@ -384,20 +384,24 @@ chk_job_request(char *jobid, struct batch_request *preq, int *rc, int *err)
 			jobid, msg_unkjobid);
 		if (err != NULL)
 			*err = PBSE_UNKJOBID;
-		req_reject(PBSE_UNKJOBID, 0, preq);
+
+		if (preq->rq_type != PBS_BATCH_DeleteJobList)
```

Comment:
you are doing this 5 times in this function, how about adding a goto at the bottom of the function which will do this check, reject if not PBS_BATCH_DeleteJobList and return NULL?

Reply:
The error code for each req_reject() is different, hence we can't have a common code in the goto function.